### PR TITLE
fix(images): extract third-party image paths into top-level variables

### PR DESCRIPTION
**Pull request by oligraw**
Instead of whatever the latest is at install time

### CODE REVIEW

File: nms_stack/nms_cli/nms_stack/roles/nms/templates/docker-compose.yml [hunk 1]
```diff
@@ -48,7 +48,7 @@ services:
         max-file: "10"
 
   grafana:
-    image: grafana/grafana:latest
+    image: grafana/grafana:8.5.3
```
Comment (aclave1) [inline nms_stack/nms_cli/nms_stack/roles/nms/templates/docker-compose.yml:51]: I think this should be `{{grafana_image}}` so it uses the variable.

**Review by aclave1 — CHANGES_REQUESTED** [review]

File: nms_stack/nms_cli/nms_stack/roles/nms/templates/docker-compose.yml (hunk 1 above)
Comment (oligraw) [reply nms_stack/nms_cli/nms_stack/roles/nms/templates/docker-compose.yml:51]: IIUC the variable only exists in k8_stack? They seem to be done inline for nms_stack (same with efk images)

Comment (oligraw) [reply nms_stack/nms_cli/nms_stack/roles/nms/templates/docker-compose.yml]: PR updated with image paths extracted

**Comment by elludraon** [comment]
Please re-open if you are able to sign the CLA. Thanks!

**Pull request closed without merging**